src/csync/csync.h

```diff
@@ -193,26 +193,19 @@ struct OCSYNC_EXPORT csync_file_stat_s {
 /**
  * csync handle
  */
-typedef struct csync_s CSYNC;
+using CSYNC = struct csync_s;
```

Comment:
This one looks more like an attempt to keep some of the structure details private... I know it could be done with just a forward declare but I prefer to keep the original intent there.